packages/delimcc_of_fxhandler/delimcc_of_fxhandler.0.1/opam

```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Delimited control primitives on top of effect handlers"
+description: "Delimited control primitives on top of effect handlers"
+maintainer: ["KC Sivaramakrishnan"]
+authors: ["KC Sivaramakrishnan"]
+license: "ISC"
+tags: ["effect handlers" "delimited control"]
+homepage: "https://github.com/kayceesrk/delimcc_of_fxhandler"
+bug-reports: "https://github.com/kayceesrk/delimcc_of_fxhandler/issues"
+depends: [
+  "ocaml" {>= "5.0.0"}
+  "multicont"
+  "dune" {>= "3.7"}
+  "odoc" {with-doc}
+]
+build: [
```

Comment:
```suggestion
conflicts: ["ocaml-option-bytecode-only"]
build: [
```

Is this intended to work with bytecode-only backends?  This option above should avoid this being selected on those (arm32, etc)

Reply:
Thanks for the suggestion.